app.py

```diff
@@ -20,6 +21,21 @@
 if __name__ == '__main__':
     commandLineArgs = command_line_args.getCommandLineArgs()
 
+    # Load database config
+    # notify user if the app will be using test or prod db
+    if (commandLineArgs.db == 'prod'):
+        print('Using production database...')
+        db_yaml = utils.get_yaml('data/database/config_prod.yml')
+    else:
+        print('Using local database...')
```

Comment:
Same with comment about db naming above; should this be the same db selection (dev or test) specified on the command line?